PROCESS.md

```diff
@@ -0,0 +1,166 @@
+- Proposals may also be added to the Flux Dev weekly meetings agenda, as a good avenue for making progress on a decision <https://lists.cncf.io/g/cncf-flux-dev/calendar>.
+
+## Project Processes
+
+### Inactivity
+<!--TODO: project leads to fill in exact details for how you measure inactivity for your project-->
+
+It is important for contributors to be and stay active to set an example and show commitment to the project.
+Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project.
+
+Inactivity is to be defined by the [core maintainers] team.
+
+Consequences of being inactive include:
+
+- Involuntary Removal
+- Being moved to Emeritus status
+
+### Involuntary Removal
+
+Involuntary removal of a contributor happens when responsibilities and requirements aren't being met.
+This may include repeated pattern of inactivity, extended period of inactivity, and/or a violation of the Code of Conduct.
+This process is important because it protects the community and its deliverables while also opens up opportunities for new contributors to step in.
+
+Involuntary removal is handled by the [core maintainers] team. Removing a Repository Maintainer or Core Maintainer for any reason other than inactivity is a [Supermajority decision](#supermajority-decisions).
+
+### Stepping Down/Emeritus Process
+
+If and when contributors' commitment levels change, contributors can consider stepping down (moving down a role) vs moving to emeritus status (completely stepping away from the project).
+
+Please reach out to the [core maintainers] team to discuss this process.
+
+### Stepping Back Into a Role
+
+If and when someone is available to step back into a previous contributor role, this is something that can be arranged and considered by the [core maintainers] team.
+
+Please reach out to the [core maintainers] team to discuss this process.
+
+### Licensing changes
+
+Licensing and intellectual property changes is a [Supermajority decision](#supermajority-decisions). To be made by the [core maintainers].
+
+### Governance changes
+
+[core maintainers] decide on material changes to the Governance document.
+ This is a [Supermajority decision](#supermajority-decisions).
+
+- Note: editorial changes to governance may be made by lazy consensus, unless challenged.
+  These are changes which fix spelling or grammar, update work affiliation or similar, update style or reflect an outside and obvious reality.
+  They do not change the intention or meaning of anything in this document.
+
+### Contact
+
+For inquiries, please reach out to: `@fluxcd/core-maintainers` on GitHub.
+
+### Requesting new repositories / bots / GitHub applications / etc
+
+File an issue in the `fluxcd/community` repository and contact the `@fluxcd/org-admins` team to fulfil your request.
```

Comment:
```suggestion
File an issue in the `fluxcd/community` repository and contact the `@fluxcd/org-admins` team to fulfill your request.
```

Reply:
I believe this is due to British English and US English spelling differences 🇬🇧 🙃 🇺🇸

Reply:
But still, no harm, so merging. Thanks @kingdonb 😊